app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -117,6 +117,17 @@ def hash_for_pdf
                                                  ]))
       end
 
+      # line 12
+      if @xml_document.at("Exemptions DependAttendCollege")
+        count = @xml_document.at("Exemptions DependAttendCollege").text.to_i
+        digits = count.digits
+        answers[:undefined_14] = digits[0]
+        answers[:'x  1000_4'] = count * 1_000
+        if digits.length.positive?
+          answers[:Text49] = digits[1]
+        end
```

Comment:
[dust|sand] also on my PR I have [this change](https://github.com/codeforamerica/vita-min/pull/5014/files#diff-2062992eaee6859ddbb1cc1b5096eeae8efbc7e474aa5ee72ef286e35618a239R575-R586) to `insert_digits_into_fields` so that it works for non-decimals too. if you want to do it this way and i'll just remember to refactor in my PR that is one way we could do this?

Reply:
Oh neat! I don't have strong feelings but I think using your modification is slightly nicer. How should it work logistically? Are you suggesting I keep it this way and you refactor this code with your new pattern after I merge? Or something else?

Reply:
either way is fine with me, but since my PR already has the new function introduced, maybe let's just keep that and i'll do the refactor in that PR after this is merged? it will just be replacing the lines here with `digits` with a call to `insert_digits_into_fields`